lib/index.d.ts

```diff
@@ -25,7 +25,13 @@ export enum SIGNATURE_TYPES {
     CANONICAL_TEXT = 1
 }
 
-export type OpenPGPKey = key.Key;
+// type defined in OpenPGP is not complete
+export interface OpenPGPKey extends key.Key {
+    users?: ({ userId?: { userid?: string } })[];
+}
```

Comment:
Ah, that's nice! But do you know if all the types that we have in openpgp already are there? But yeah, we can use that, and at max fill missing types through pmcrypto. I'll open a PR there.

The type for `userid` is obviously not needed here, but it's needed in some other PRs I'm preparing for mail. I thought it would be ok to squeeze both the SHA1 thing and the type (since it's only one) in the same PR, but I'm gonna remove it hoping that they accept the change in FlowCrypt

Reply:
Because of cross-dependencies with other projects, I think it's better to keep the types from openpgp